CHANGELOG.md

```diff
@@ -5,7 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased](https://github.com/simplepie/simplepie/compare/1.8.0...master)
```

Comment:
We should keep the `Unreleased` section for future notes and just update the tag from 1.8.0 to 1.8.1

See also https://keepachangelog.com/en/1.1.0/#effort

Reply:
I find it weird to have stubs like this in a final release. I could add it in a separate commit and then tag the first commit.

Or maybe it does not matter.

Reply:
For the newest release you are right. But with more time this section becomse more important. One user with the e.g. 1.7.0 release has now a link to all new changes that one will get with an update.

See https://github.com/simplepie/simplepie/blob/1.7.0/CHANGELOG.md

Reply:
I do not see much of a point – you typically update to a specific version, so you would check the changelog for the version you are updating to and if you want to see commits, you would look at the diffs version by version. If you really want all commits at once, you can modify the URL or use `git log` but that should be rare. Either way, seeing all commits until `HEAD` will give you extra commits past the last release, which are not relevant unless you are updating to `HEAD`.

Also, once next version rolls in, the “Unreleased” link label is no longer accurate.

But it is not that important, it is only mildly confusing so until https://github.com/olivierlacan/keep-a-changelog/discussions/517 is resolved, I am fine with following what [keep-a-changelog project does](https://github.com/olivierlacan/keep-a-changelog/blob/v1.1.1/CHANGELOG.md) and not bother removing it.

Reply:
If it helps, in other projects, I often see the "Unreleased" header (and link) to remain on releases, but accompanied by one extra sentence underneath the header: _Nothing yet._